packages/graphback/src/buildGraphbackAPI.ts

```diff
@@ -73,6 +75,7 @@ function createServices(models: ModelDefinition[], createService: Function, crea
 export function buildGraphbackAPI(model: string | GraphQLSchema, config: GraphbackAPIConfig): GraphbackAPI {
   const schemaPlugins: GraphbackPlugin[] = [
     new SchemaCRUDPlugin,
+    new ResolversCRUDPlugin,
```

Comment:
Actually don't think this is a good idea, because it means there is no way to NOT use Graphback resolvers, but I like the idea of the user not having to import the resolver plugin